readthedocs_ext/readthedocs.py

```diff
@@ -32,6 +32,7 @@
     "_static/doctools.js": "%sjavascript/doctools.js",
```

Comment:
I think actually we should probably _stop_ hosting these on our servers, and let the builds include their own version. This is another place where we are weirdly coupling one version of JS across all versions of the builds. I think this has already caused bugs in the past. It will mean we are serving those files from the webs, instead of our assets domain, but I think it's probably better, similar to the work we did with unbundling the asset-hosted CSS files.